test/jdk/java/time/test/java/time/TestClock_Tick.java

```diff
@@ -73,6 +73,8 @@
 
 /**
  * Test tick clock.
+ *
+ * @bug 8310232
```

Comment:
Did you mean the put the "@bug" here? If you did then I assume you'll need to add the original bug ID too as TestClock_Tick wasn't created for 8310232.

Reply:
I meant to put the bug id here, but you are right that it would confuse readers as if it is the bug that created this regression test. Will remove it and leave the bug id only for the added test.